contentrepo/settings/dev.py

```diff
@@ -7,7 +7,12 @@
 SECRET_KEY = os.environ.get("SECRET_KEY", DEFAULT_SECRET_KEY)
 
 # SECURITY WARNING: define the correct hosts in production!
-ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "localhost")
+ALLOWED_HOSTS = os.environ.get(
+    "ALLOWED_HOSTS",
+    [
+        "localhost",
+    ],
+)
```

Comment:
I don't think this will work very well, since if we every define `ALLOWED_HOSTS`, then we'll get a string again instead of a list. We should probably just use `env.list` like in the production config, but with the `localhost` default